src/app/containers/SocialEmbed/index.jsx

```diff
@@ -82,14 +83,16 @@ const SocialEmbedContainer = ({ blocks }) => {
             caption={caption}
           />
         ) : (
-          <CanonicalSocialEmbed
-            provider={provider}
-            service={service}
-            oEmbed={oEmbed}
-            fallback={fallback}
-            skipLink={skipLink}
-            caption={caption}
-          />
+          <Lazyload once>
```

Comment:
Would we expect any tests to be updated as a result of this?

Reply:
Snapshots will probably need updating, just letting CI run to find out